src/components/tabs/README.md

```diff
@@ -111,9 +111,49 @@ variant.
 <!-- b-tabs-pills.vue -->
 ```
 
+## Fill and justify
+
+Force your `<b-tabs>` content to extend the full available width.
+
+### Fill
+
+To proportionately fill all available space with your `<b-tab>` components, set the `fill` prop.
+Notice that all horizontal space is occupied, but not every nav item has the same width.
+
+```html
+<div>
+  <b-tabs content-class="mt-3" fill>
+    <b-tab title="First" active><p>I'm the first tab</p></b-tab>
+    <b-tab title="Second"><p>I'm the second tab</p></b-tab>
+    <b-tab title="Very, very long title"><p>I'm the tab with the very, very long title</p></b-tab>
+    <b-tab title="Disabled" disabled><p>I'm a disabled tab!</p></b-tab>
+  </b-tabs>
+</div>
+
+<!-- b-tabs-fill.vue -->
+```
+
+### Justified
+
+For equal-width elements, set prop `justified` instead. All horizontal space will be occupied by nav
+items, but unlike `fill` above, every `<b-tab>` will be the same width.
+
+```html
+<div>
+  <b-tabs content-class="mt-3" justified>
```

Comment:
Should this allow other options, such as centered tab buttons, and right justified?

Maybe it should be called `align`, which accepts values like `left` (the default, which needs no classes), `right`, `center`, and `justified` ?

Reply:
What about leaving it `justified` and accept both a `Boolean` and `String` (e.g. `justified="right"`)?

Reply:
Hmmm that might be an option.

You would need to check for empty string `''` to also mean boolean `true` though.

Reply:
Also, these options would only be applicable when the controls are on top or bottom of the tabs component (for vertical tabs it wouldn't make sense)